test/docker/docker-compose.yml

```diff
@@ -0,0 +1,97 @@
+version: "3"
```

Comment:
Can you please fix versions on images in this file?

Reply:
https://github.com/stellar/typescript-wallet-sdk/pull/74/commits/706c890ed3a1b5899d1a5ecdee3892c04930f664